bindings/pyroot/ROOT.py

```diff
@@ -274,29 +274,59 @@ def _TTreeAsMatrix(self, columns=None, dtype="double"):
     # Get all columns of the tree if no columns are specified
```

Comment:
Should we throw or just return an empty numpy array?

Reply:
imho silently returning an empty array is a bit evil

Reply:
Just checked with pandas dataframe:
```
import pandas as pd
d = {'a' :pd.Series([])}
df = pd.DataFrame(d)
df.as_matrix()
Out: array([], shape=(0, 1), dtype=float64)
```

Reply:
To be fully clear, my comment was targeting `if self.GetEntries() == 0`